extensions/hibernate-reactive/deployment/pom.xml

```diff
@@ -20,6 +20,28 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-hibernate-orm-deployment</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-narayana-jta-deployment</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-narayana-jta</artifactId>
+                </exclusion>
+                <exclusion>
```

Comment:
Doesn't this dependency cause the TRANSACTIONS capability to be available at build time, and therefore  `QuarkusJtaPlatform` used? Does this need to be excluded from the deps?

Reply:
Hadn't noticed this was an exclusion!